src/ResourceManager/DevTestLabs/Commands.DevTestLabs/Commands/Get/GetAzureRmDtlAllowedVMSizesPolicy.cs

```diff
@@ -26,7 +26,6 @@ public class GetAzureRmDtlAllowedVMSizesPolicy : DevTestLabsCmdletBase
         public override void ExecuteCmdlet()
         {
             var policy = DataServiceClient.Policies.Get(
-                ResourceGroupName,
```

Comment:
Have you manually verified that all these cmdlets now work with the changes?  Since you don't have any tests associated with this module, I would like to get confirmation that all of these cmdlets are now working as intended.

Reply:
Hi Maddie,
I have manually verified all these cmdlets now work.

Reply:
Great, I'll kick off an on demand run and merge as soon as it passes.